docs/__tests__/schema.test.js

```diff
@@ -10,7 +10,7 @@ function validateSchemaDocumentation(formBuilder, schemaDocPath) {
     const docContents = fs.readFileSync(schemaDocPath, 'utf8');
 
     describe(`${form.title} schema documentation`, function () {
-        test.each(Object.keys(form.schema.describe().children))(
+        test.each(Object.keys(form.schema.describe().keys))(
```

Comment:
`schema.describe` has changed quite a bit. For our purpose this is the only change for now, but in theory means we could do more with it.